governance/roles.md

```diff
@@ -92,23 +93,37 @@ TLs are assigned by CNCF Technical Oversight Committee
 
 ## Role of project leads
 
-Project Leads will lead larger streams of work that require sustained
-effort and coordination.  In some cases, these projects remain ongoing.
+Project Leads lead specific projects where a deliverable or set of deliverables
+have been defined within the issue.
 
 Project Leads are nominated and approved by the following process:
-  1. Project Lead actively participates in the group, making a proposal or
-  volunteering to take on a project that has been prioritized by the group
+  1. Project Lead actively participates in the group, initiates the proposal or
+  volunteers to take on a project that has been prioritized by the group
   1. A Chair or TL nominates a candidate.
-  1. The nomination is communicated via a pull request annotating the list of members in the 
-  [SIG README](/README.md) with a link to the issue tracking the project. 
+  1. The nomination is communicated on the issue tracking the project 
   The nomination is typically open for a week (but may be shorter with LGTM of at least two Chairs).
-  1. Members are encouraged to review the tracking issue and work together
-  to ensure that the Project Lead is set up for success or suggest alternatives.
+  1. The issue is updated with the assignment of the project lead as "assignee"
+     alongside the SIG Leadership member.
 
 Depending on the expected length of the project, Project Leads may be assigned
 as OWNERS of the project directory in this repository to help facilitate and moderate new content to the
 project.
 
+## Role of team leads
+
+Team Leads lead larger streams of work that require sustained effort and 
+coordination. In some cases, these remain ongoing.
+
+Team Leads are nominated and approved by the following process:
+  1. Team Lead actively participates in the group, initiates the proposal or
+  volunteers to lead a stream that has been prioritized by the group
+  1. A Chair or TL nominates a candidate.
+  1. The nomination is communicated via a pull request annotating the list of members in the
+  [SIG README](/README.md) with a link to the issue tracking the stream of work.
+  The nomination is typically open for a week (but may be shorter with LGTM of at least two Chairs).
+  1. Members are encouraged to review the tracking issue and work together
```

Comment:
I wasn't thinking there would be a single tracking issue for a team.  I think that would be confusing.  Rather team has a sub-directory with a README about how to get involved.